tests/playwright/utils/auth.ts

```diff
@@ -0,0 +1,320 @@
+import {
+  CognitoUserPool,
+  CognitoUser,
+  AuthenticationDetails,
+  CognitoUserSession,
+  IAuthenticationCallback,
+} from "amazon-cognito-identity-js";
```

Comment:
There's a note on the npm page encouraging amplify v6 to be used.  I'm not sure if that's possible for your use case.
https://www.npmjs.com/package/amazon-cognito-identity-js